admin/onpage/class-onpage-option.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * This class handles the data for the option where the OnPage.org data is stored.
+ */
+class WPSEO_OnPage_Option {
+
+	/**
+	 * @var array The OnPage.org option stored in the database.
+	 */
+	private $onpage_option;
+
+	/**
+	 * Setting the object by setting the properties
+	 */
+	public function __construct() {
+		$this->onpage_option = $this->get_option();
+	}
+
+	/**
+	 * Getting a value from the OnPage.org option
+	 *
+	 * @param string $option_key The needed key from the option.
+	 * @param string $default    The default value, when the key doesn't exist.
+	 *
+	 * @return string
+	 */
+	public function get( $option_key, $default = '' ) {
+		if ( array_key_exists( $option_key, $this->onpage_option ) ) {
+			return $this->onpage_option[ $option_key ];
+		}
+
+		return $default;
+	}
+
+	/**
+	 * Setting a value to the OnPage.org option
+	 *
+	 * @param string $option_key   The key which will be stored.
+	 * @param string $option_value The value that has to be stored.
+	 */
+	public function set( $option_key, $option_value ) {
+		$this->onpage_option[ $option_key ] = $option_value;
+
+		$this->save_option();
+	}
+
+	/**
+	 * Check if the last fetch is within the time of 60 minutes
+	 *
+	 * @return bool
+	 */
+	public function can_fetch() {
```

Comment:
`can_fetch` sounds like it means we are checking if it is possible to fetch, but instead it returns if we should fetch. We should rename it to `should_be_fetched`, or something better.